schema/index.ts

```diff
@@ -0,0 +1,4 @@
+export * from "./issues";
```

Comment:
@ssk090 
As you are exporting all the schema from index file, what do you think about updating the existing schema imports to point to this index file and not individual schema files?

Reply:
yes, I agree with that. My point was could you update the existing imports to import for the index file? Because currently in some places we are directly importing from file.

Reply:
yes, as part of this commit: https://github.com/aadeshkulkarni/first-issues/pull/19/commits/6311d75734f6947a4f80623881c07881eb4814db i have made the changes. Let me know if i missed something